### PR TITLE
Feature/504c migrate typography

### PR DESCRIPTION
**Pull request by calumryan**
@isratc please review the changes (not able to merge yet awaiting help).

### CODE REVIEW

**Comment by isratc** [comment]
@calumryan
**common.scss** line 244
Replace with mixing box shadow

**Forms.scss**: multiple mixins to be added, have a skim through the page.

**private-notice.scss**: Replace font with GOV Font

**partials/banner.scss/**: Replace font with GOV Font and add Colour variables

**partials/dxw.digest.scss/**: Replace font with GOV Font and add mixins

**Comment by isratc** [comment]
@RobjS 
Ive comparing my commit that i did in `templates/ ` On the original branch: [https://github.com/dxw/govuk-blogs/tree/feature/504-main-theme-editable-areas-to-be-migrated-and-general-typography](url)
Under @calumryan "resolve conflict" commit (which have somehow took my changes) my edits are not showing in the history of this new branch?

**However the changes seem to be in the branch!** 
It's just not showing as a commit in the history of commits of this new branch I had to compare it with the original branch to make sure that the changes were there?

```
templates/content-single.php
templates/entry-meta.php
templates/featured.php
templates/header.php
templates/sidebar.php

**Only one showing commit in this PR:**
templates/entry-meta.php
```

**Comment by isratc** [comment]
@calumryan I have fixed the latest merge conflict

**Comment by calumryan** [comment]
@isratc I have added these changes to mixins and variables

File: assets/css/common/mixins.scss [hunk 1]
```diff
@@ -40,6 +40,20 @@ $is-ie: false !default;
...
+  -webkit-box-sizing: $sizing;
+  -moz-box-sizing:    $sizing;
+  box-sizing:         $sizing;
+}
+
+@mixin transform($transform) {
+  -moz-transform: $transform; // Firefox 15 compatibility
+  -o-transform: $transform; // Opera 12.0 compatibility
+  -webkit-transform: $transform; // Safari 8 compatibility
+  -ms-transform: $transform; // IE9 compatibility
```
Comment (mallorydxw) [inline assets/css/common/mixins.scss:53]: We don't worry about any of those browsers: https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices

File: assets/css/admin.scss [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
+$govuk-images-path: "../build/node_modules/govuk-frontend/govuk/assets/images/";
+$govuk-fonts-path: "../build/node_modules/govuk-frontend/govuk/assets/fonts/";
+@import "govuk-frontend/govuk/all";
+
 .hallo-editor {
   @import "common/variables";
   @import "common/mixins";
   @import "common/common";
-  @extend .entry-content;
+  @extend .entry-content !optional;
```
Comment (mallorydxw) [inline assets/css/admin.scss:10]: TIL about [this feature](https://sass-lang.com/documentation/at-rules/extend#mandatory-and-optional-extends). Thanks!

But can I ask why we're using it here? Does this need to be optional for the sake of the child theme?

**Comment by isratc** [comment]
I am still seeing some style sheets using colours that is not a variable should be a variable, for a example "red"
should be using govuk-colour("red");`

https://design-system.service.gov.uk/styles/colour/

File: assets/css/admin.scss (hunk 2 above)
Comment (calumryan) [reply assets/css/admin.scss]: Removed again optional, it was removed in a previous branch but not this one.

**Comment by calumryan** [comment]
I've updated the colour variables but for any that are hypenated as these don't seem to compile. Temporarily commented these adjacent to the property for reference.

**Comment by isratc** [comment]
This PR can be merged in as we will be looking at the colour palette in a separate card at as it needs further investigation with a dev.

**Pull request merged**